src/Calculator/EquationStylePanelControl.xaml.cpp

```diff
@@ -145,61 +145,81 @@ String ^ EquationStylePanelControl::GetColorAutomationName(Brush ^ brush)
     auto resourceLoader = AppResourceProvider::GetInstance();
     auto color = static_cast<SolidColorBrush ^>(brush);
 
-    if (color == safe_cast<SolidColorBrush ^>(Application::Current->Resources->Lookup(L"EquationBrush1")))
+    auto lightDictionary = static_cast<ResourceDictionary ^>(Application::Current->Resources->ThemeDictionaries->Lookup(L"Light"));
```

Comment:
Is it intended for high contrast to have all of the labels be Black also? Maybe Color 1, Color 2, etc might be better?